gsw/gibbs/library.py

```diff
@@ -140,7 +140,7 @@ def _delta_SA(self, p, lon, lat):
         mask_in = np.ma.mask_or(np.ma.getmask(p), np.ma.getmask(lon))
         mask_in = np.ma.mask_or(mask_in, np.ma.getmask(lat))
         p, lon, lat = [np.ma.filled(a, 0).astype(float) for a in (p, lon, lat)]
-        p, lon, lat = np.broadcast_arrays(p, lon, lat)
+        p, lon, lat = np.broadcast_arrays(p, lon, lat, subok=True)
```

Comment:
Actually, the "subok=True" option doesn't make sense in any place like this where we can be sure the arguments are ndarrays--using np.ma.filled guarantees that.  Although it doesn't hurt anything, it makes the code less readable.